### ISSUE TITLE
Add group outing plans?

### ISSUE THREAD

**Issue by ndenny** [pycascades/welcome-wagon-2019#2]

**Comment by hlandrus** [comment]
Is there a desire to add this still? I had some ideas but haven't had a chance to make it well formed. List of ideas karaoke, climbing, museums, find music shows, shopping options, libraries, … and more?

**Comment by ndenny** [comment]
Yeah, this is currently my greatest failure… I had an idea of suggesting people go to the Living Computer Museum, but given we’re 4 days out and I’ve got as far as finding out that they’re closed on Mondays, I fear that boat has sailed.

Still, no harm in adding more content, even if it is very high level. Even if you create the headings, we can always accept PRs with more detail.
